lecli/cli.py

```diff
@@ -326,9 +327,93 @@ def deleteuser(userkey):
 @cli.command()
 def getowner():
     """Get account owner details"""
-
     user_api.get_owner()
 
 
+@cli.command()
+@click.option('-n', '--name', type=click.STRING, default=None,
+              help='Name of new log')
+@click.option('-f', '--filename', type=click.STRING, default=None,
+              help='File containing json representation of log object')
+def createlog(name, filename):
+    """Create a log with the provided name and details.
+    Will use JSON file first if both name and file are provided"""
+
+    if filename is not None:
+        with open(filename) as json_data:
+            # Open file and load as JSON
+            try:
+                params = json.load(json_data)
+                log_api.create_log(None, params)
+            except ValueError as error:
+                print error
+                exit(1)
+    elif name is not None:
+        log_api.create_log(name, None)
+    else:
+        click.echo('Example usage: lecli createlog -n \'new_log_name\'')
```

Comment:
how about `click.echo("Example usage: lecli createlog -n 'new_log_name'")` however, why do we need to have single quotes around `new_log_name`